### PR TITLE
add(templates): Easy Tailwind

### PR DESCRIPTION
**Pull request by sidchaudhary04**
| Name                 | Link                 |
| -------------------- | -------------------- |
| _Easy Tailwind_ | _https://easytailwind.now.sh |

_Resource description here_

---

- [x] My item is in the right category
- [x] My item is logically grouped below similar items
- [x] My item's name and description respects the conventions of the list
- [x] My item is awesome
- [x] I have read and followed the [contribution guidelines](.github/CONTRIBUTING.md)

### CODE REVIEW

**Comment by innocenzi** [comment]
I'm not against pulling this in, but: 
- It looks very not polished (no title, a lot of lorem ipsum)
- You didn't respect the guidelines (should be at the bottom of its emoji group, no PR title nor description)

I don't mean to be rude or anything, but the list is for awesome content and it looks like you just finished the main content of your product. If it were only me I'd wait for you to polish it a bit more and I'd happily merge it.

Thoughts @joshmanders?

**Comment by sidchaudhary04** [comment]
Thank you for your helpful feedback and I am sorry about the ordering on the list. 
What I don't understand is the "lorem ispum" you refer to. Are you referring to the example text on the template that I am selling? I am open to suggestions and feedback.

**Comment by innocenzi** [comment]
Oh, I think I just confused myself. I clicked Preview and I thought it was the product page since there were the same pricings, so I was referring to the lorem ipsums in Blog and About. 

My bad on this one. Makes more sense. I suggest you try to make more obvious that we're visiting the actual product then. 

I'll wait the approval of @joshmanders before merging in.

File: README.md
Comment (innocenzi) [inline README.md:212]:
```suggestion
- 📁 [Tailwind Made](https://tailwindmade.com/) - Paid, developer-friendly templates made with Tailwind CSS.
- 📁 [EasyTailwind](https://easytailwind.now.sh) - Tailwind CSS templates handcrafted.
```

Comment (innocenzi) [reply README.md:212]: Can you put your item at below the previous one, and maybe make a more descriptive description?

**Review by innocenzi — CHANGES_REQUESTED** [review]

File: README.md
Comment (sidchaudhary04) [reply README.md:212]: Will do

**Comment by sidchaudhary04** [comment]
Changed the description, hope it is okay

**Review by innocenzi — APPROVED** [review]

**Comment by joshmanders** [comment]
I was hesitant earlier because it was so scarce, but if you are ok with it @innocenzi, then it gets my approval.

**Pull request merged**
